src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -393,9 +446,103 @@ public static ConstantSet getConstants(String classname, ConstantSet result) {
               // Push a value from the constant pool. We'll get these
               // values when processing the constant pool itself.
             case Const.LDC:
+              {
+                LDC ldcInstruction = (LDC) inst;
```

Comment:
It seems to me that there are two separate concerns in this pull request.

 * One is obtaining more constants by looking at instructions such as `LDC`.
 * The other is changing the weighting scheme.

Can you make two different pull requests, one for each feature?  That will be easier to review, and it can permit them to be enabled and disabled individually.